include/seastar/core/io_priority_class.hh

```diff
@@ -92,6 +92,8 @@ private:
     static constexpr unsigned _max_classes = 2048;
     static std::mutex _register_lock;
     static std::array<class_info, _max_classes> _infos;
+
+    friend std::tuple<unsigned, sstring> get_class_info(io_priority_class_id pc);
```

Comment:
Better to use a struct so the return is self-documenting.

Reply:
I kept it tuple in the next version as per your next comment

Reply:
Don't understand. My next comment says that destructuring would still work with a struct as it does with a tuple.

Reply:
Ah... I misunderstood your next comment in a way "ah, so you 'self-document' the return values here, so OK".
In fact, this helper is local to io_queue.cc and the above friendship goes away with API-level 6

Reply:
ok